README.md

```diff
@@ -61,23 +61,16 @@ Things not covered: tokens and secrets for Patreon integration, Slack integratio
         [register a new application](https://dev.twitch.tv/console/apps/create). Set the redirect URI to `http://localhost:5000/login`. 
 
 
- 3. Write a `data.json` file. Basic template:
```

Comment:
A quick `grep` shows that migrations `affc03cb46f5_game_data`, `cddcdf06d9f9_storm` and `a6854cd2e5f1_users` access `data.json` but don't guard against it not existing. Those should be fixed as well.